src/org/thoughtcrime/securesms/service/RegistrationService.java

```diff
@@ -260,17 +260,16 @@ private void handleCommonRegistration(TextSecureAccountManager accountManager, S
     DatabaseFactory.getIdentityDatabase(this).saveIdentity(self.getRecipientId(), identityKey.getPublicKey());
     DirectoryHelper.refreshDirectory(this, accountManager, number);
 
-    //if (GooglePlayServicesUtil.isGooglePlayServicesAvailable(this) == ConnectionResult.SUCCESS &&
-    //    !BuildConfig.FORCE_WEBSOCKETS)
-    //{
-    //  RedPhoneAccountManager redPhoneAccountManager = new RedPhoneAccountManager(BuildConfig.REDPHONE_MASTER_URL,
-    //                                                                           new RedPhoneTrustStore(this),
-    //                                                                           number, password);
-    //  String verificationToken = accountManager.getAccountVerificationToken();
-    //  redPhoneAccountManager.createAccount(verificationToken, new RedPhoneAccountAttributes(signalingKey,
-    //          TextSecurePreferences.getGcmRegistrationId(this)));
-    //}
-    accountManager.getAccountVerificationToken();
+    String verificationToken = accountManager.getAccountVerificationToken();
```

Comment:
IMHO it would be a good thing to have a coordinated place for LibreSignal, I don't think users want to follow different custom builds and releases. So @JavaJens Could we get this integrated here at first?